ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -133,6 +136,18 @@ OnboardingStackView {
         }
     }
 
+    Connections {
+        target: root
+
+        function onReencryptingDatabaseChanged() {
+            if (root.reencryptingDatabase) {
+                root.push(reencryptingDatabaseScreen, StackView.Immediate)
+            } else {
+                root.pop(StackView.Immediate)
```

Comment:
It's quite risky behavior, especially when we modify the stack internally and also from outside. We need to be sure that one we are going to pop is `reencryptingDatabaseScreen`. What if in the meantime some other page is pushed?

Reply:
This screen was removed now, according to the design 👌